app/src/views/internships/index.vue

```diff
@@ -185,17 +188,17 @@ export default class extends Vue {
     title: [{ required: true, message: 'title is required', trigger: 'blur' }],
   };
   private downloadLoading = false;
-  private tempBusinessData = defaultBusinessData;
+  private tempInternshipData = defaultInternshipData;
 
   public created() {
     this.getList();
   }
 
   private getList() {
     this.listLoading = true;
-    getBusinesses(this.listQuery).then((data: any) => {
-      this.list = data;
-      this.total = data.length;
+    getInternships(this.listQuery).then((res: any) => {
+      this.list = res.data;
```

Comment:
``` javascript
this.list = res ? res.data : []
this.total = res ? res.max : 0
```

This code allow you to handle case when API response 204 - No content
See [api - internships](https://ryshu.github.io/enib-internships/docs/api-internships) for more detail